public/script.js

```diff
@@ -2,18 +2,45 @@ let CONSTANTS;
 const STATE = {
   stop: true,
   rate: 100,
-  current: moment().year(1990).month(0).day(1),
-  start: moment().year(1990).month(0).day(1),
-  end: moment().year(2020).month(11).day(31),
+  current: moment(),
+  start: moment(),
+  end: moment(),
   extCon: 'extent',
```

Comment:
As a note for the meeting, I saw your comment about changing these names and made those commits (they should be on master now), but I left it how it was here to merge those changes eventually.